src/main/cpp/alias/Alias.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Alias.h
+ *
+ *  Created on: Mar 11, 2017
+ *      Author: cedric
+ */
+#ifndef ALIAS_H_
+#define ALIAS_H_
+
+#include <util/Random.h>
+#include <vector>
+namespace stride{
+namespace alias{
+/**
+ * Class used to generate random numbers according to Vose's Alias Method.
+ * Used the algorithm on http://keithschwarz.com/darts-dice-coins/
+ * It uses an strides Random as random number generator.
+ */
+class Alias {
+public:
+	///Constructor. Uses a random seed
+	Alias(const std::vector<double> probabilities);
```

Comment:
Slapping just `const` on a parameter doesn't *actually* mean anything.

Did you mean to write `const std::vector<double>&`? That's also a whole lot more efficient because it avoids copying the whole `std::vector<double>` on every call to `Alias::Alias`.